portality/api/v1/crud/applications.py

```diff
@@ -51,32 +58,78 @@ def create(cls, data, account, dry_run=False):
         # if that works, convert it to a Suggestion object
         ap = ia.to_application_model()
 
-        # if the caller set the id, created_date or last_updated, then can the data, and apply our
-        # own values (note that last_updated will get overwritten anyway)
-        ap.set_id()
-        ap.set_created()
-
         # now augment the suggestion object with all the additional information it requires
         #
         # suggester name and email from the user account
         ap.set_suggester(account.name, account.email)
 
-        # suggested_on right now
-        ap.suggested_on = datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
-
-        # initial application status for workflow
-        ap.set_application_status('pending')
-
-        # set the owner to the current account
-        ap.set_owner(account.id)
-
         # they are not allowed to set "subject"
         ap.bibjson().remove_subjects()
 
-        # finally save the new application, and return to the caller
-        if not dry_run:
-            ap.save()
-        return ap
+        # if this is an update request on an existing journal
```

Comment:
marker for me, reviewed down to here. These need to be reviewed carefully (not that the rest don't), they are a significant refactor of the API code close to the surface